_studio/mfx_lib/encode_hw/hevc/agnostic/g9/hevcehw_g9_ext_brc.cpp

```diff
@@ -494,7 +494,13 @@ void ExtBRC::QueryTask(const FeatureBlocks& /*blocks*/, TPushQT Push)
             break;
         case MFX_BRC_PANIC_BIG_FRAME:
             task.bSkip = true;
+#ifndef ANDROID
```

Comment:
There is nothing specific to Android in this code. I think it's better to remove ANDROID define here and fix clang warning for all OSes. For example, something like that:
```
        case MFX_BRC_PANIC_BIG_FRAME:
            task.bSkip = true;
#if __clang__
            [[clang::fallthrough]];
#endif 
        case MFX_BRC_BIG_FRAME:
#if __clang__
            [[clang::fallthrough]];
#endif
```

@mgonchar : advice whether we want more generic approach. For example, we can have something like
```
#if __cplusplus > 201703L
#define MFX_FALLTHROUGH [[fallthrough]]
#elif __clang__ 
#define MFX_FALLTHROUGH [[clang::fallthrough]]
#else 
#define MFX_FALLTHROUGH
#endif 
```

Reply:
@dvrogozh yes, your last suggestion looks fine. Let's put it to utility header (or separate header which we will remove after starting support of C++17)